doc/porting/board_porting.rst

```diff
@@ -120,3 +120,25 @@ Board
 A board implements an SoC with all its features, together with peripherals
 available on the board that differentiates the board with additional interfaces
 and features not available in the SoC.
+
+Default board configuration
+***************************
+
+When porting a board, you must provide its default configuration. This default
+configuration could be overridden, or otherwise will be the reference when an
+application, test or sample is compiled for this board.
+
+In order to provide coherency accross the various boards and ease the work of
+users providing applications that are not board specific, following guidelines
+should be followed when porting a board:
+
+- Provide pins and drivers configuration that matches the board valuable
+components, such as sensors, button, leds or communication i/f such as USB,
+Ethernet connector or Bluetooth/WiFi chip
+- When a well-known connector (arduino, 96board, ....) is present,
+configure pins to fit this connector
+- Then configure coponents that enable the use of these pins (For instance
```

Comment:
s/Then//

Reply:
- Configure components that enable the use of these pins, such as
      configuring and SPI instance for Arduino SPI.

Reply:
note the "components" typo correction pointed out by @dbkinder